JoinColony/colonyCDapp
# feat: added staged payment form

## Description

- Added staged payment form with validation, next steps will be handled in other PR
- Figma: [link](https://www.figma.com/design/5V8pr7iMwXsT9L3VAZsmUt/Colony-v5?node-id=3010-277006&t=k5oLcDv0S5S9Ij4q-4)
- Includes Create an action (form) part of scope described in https://github.com/JoinColony/colonyCDapp/issues/2669

## Testing

* Open action sidebar
* Choose Staged payment

<s>There is no issue for this yet, please check spec for more details [link](https://www.notion.so/colony/Staged-Payments-Action-f4e1dfd70e864613a5052a010b7b0116)</s>
